benchmark/utils/stencil_matrix.hpp

```diff
@@ -116,20 +116,19 @@ gko::matrix_data<ValueType, IndexType> generate_2d_stencil_box(
                             : discretization_points - i);
     };
 
-    auto flat_idx = [&](const IndexType ix, const IndexType iy) {
+    auto flat_idx = [&](const IndexType iy, const IndexType ix) {
         auto tpx = target_position(ix, positions[0]);
         auto tpy = target_position(iy, positions[1]);
```

Comment:
In the develop branch, the others don't use the same order as here to lead the wrong generation?

Reply:
Exactly, that is why the fix is needed.